insights/tools/generate_api_config.py

```diff
@@ -15,7 +15,7 @@
 import logging
 import os
 import sys
-from optparse import OptionParser
+import argparse
```

Comment:
Argparse doesn't exist in 2.6.

Reply:
We decided to resolve this by adding python-argparse to the client rpm spec file and adding a dependency in `setup.py` for virtual environments since argparse is in pypi for python >= 2.3. I've pushed the `setup.py` change, and @kylape is changing the rpm spec for the client.